web/src/lib/assets/immich-logo-inline-dark.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 28.3.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Router_Medium_x5F_Black_00000159464448132936669960000002337362428709113490_"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 792 266.25"
+	 style="enable-background:new 0 0 792 266.25;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ACCBFA;}
+	.st1{fill:#FA2921;}
+	.st2{fill:#ED79B5;}
+	.st3{fill:#FFB400;}
+	.st4{fill:#1E83F7;}
+	.st5{fill:#18C249;}
+</style>
+<g>

```

Comment:
You could also do `<g fill="currentColor">` and control the color with CSS to avoid having separate light and dark mode variants

Reply:
I tried this and then set the `text-<color>` property on the `<img/>` element, and it doesn't seem to work. I think you will need to wrap the SVG into a svelte element, like the `loading-spinner.svelte` to make it work.

Reply:
Doesn't work indeed, only if you directly include the svg in the html markup. We could still do that, or just leave it as is.